### DISCUSSION TITLE
How can I get not truncated output?

### DISCUSSION THREAD

**Discussion by nareto** [dbt-labs/dbt-core#4899] [Q&A]
Just getting started with dbt. I have some macros with this format

```
{% macro macroname %}


{% set sql %}
--- SQL code...
{% endset %}

{% set result = run_query(sql)%}
{% do result.print_table() %}

{% endmacro %}
```
which I run with `dbt run-operation`. it works fine but the output is truncated, like this for example:
```
| status               |
| -------------------- |
| Copy executed wit... |
```

is there a way to avoid that and see the full output of the query?

**Comment by michaellynton** [comment]
Yes ... try this:
`{% do result.print_table(max_column_width=50) %}`

or whatever width you want. I had a similar issue and found the arg on the [agate table docs](https://agate.readthedocs.io/en/latest/api/table.html?highlight=print_table#agate.Table.print_table)
